test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -1127,36 +1040,6 @@ define([
                     });
                 });
 
-                describe('invalid', () => {
```

Comment:
these messages won't make it past the job manager, so there isn't much point in keeping these tests in